lib/game_play.rb

```diff
@@ -0,0 +1,120 @@
+require './lib/player'
+require './lib/board'
+require './lib/cell'
+require 'pry'
+
+class GamePlay
+
+  attr_reader :human,
+              :computer
+
+  def initialize(human, computer)
+    @human = human
+    @computer = computer
+    @already_fired_upon_coordinates_human = []
+    @already_fired_upon_coordinates_computer = []
+  end
+
+
+    def display_boards
+      sleep(1)
+      puts "=============COMPUTER BOARD============="
+      puts @computer.board.render
+      puts "==============PLAYER BOARD=============="
+      puts @human.board.render(true)
+    end
+
+    def player_chooses_coordinate
+      puts "Enter the coordinate for your shot: \n"
+      input = gets.upcase.chomp
+
+
+      loop do
+
+        if @computer.board.valid_coordinate?(input) && @already_fired_upon_coordinates_human.include?(input) == false
+          @computer.board.cells[input].fire_upon
+          @already_fired_upon_coordinates_human << input
+          break
+        elsif @already_fired_upon_coordinates_human.include?(input) == true
```

Comment:
let's delete this == true here

Reply:
Just double-checked the functionality still works, and it does, so sure.